examples/using-gatsby-image/src/components/floating-image.js

```diff
@@ -0,0 +1,40 @@
+import React from "react"
+import Img from "gatsby-image"
+
+import { rhythm, options } from "../utils/typography"
+
+const FloatingImage = ({ imageMobile, imageDesktop }) => (
+  <React.Fragment>
+    <Img
+      style={{ display: `inherit` }}
+      css={{
+        marginBottom: rhythm(options.blockMarginBottom * 2),
+        marginLeft: rhythm(options.blockMarginBottom * 2),
+        marginRight: -20,
+        float: `right`,
+        "&": {
+          "@media (min-width: 500px)": {
+            display: `none`,
+          },
+        },
+      }}
+      fixed={imageMobile}
+    />
+    <Img
+      style={{ display: `inherit` }}
+      css={{
```

Comment:
We had a discussion about styling that landed on using the styled component approach for Emotion, meaning we'll want to refactor this to be:

```jsx
const GatsbyImage = styled(Img)`
	margin-bottom: ${rhythm(options.blockMarginBottom * 2)};
	/* ... */
`;
```

Reply:
Yeah, followed that and already did use `styled` for stuff I had to touch, but not everywhere else (yet). Happy to do so consistently!

Reply:
Ah, I thought this was all new code.

I'll let @amberleyromo make the call about how much we should refactor. My vote would be to just knock it out while we're in here, but she's the final word on this.

Reply:
@fk I would say if you feel it won't take too long and you feel good about it, go ahead and do it. Would be nice to have it as a good standalone example of the code style, since www will be switched over incrementally.

If not, don't stress, we can decide where it fits into priorities.

Reply:
@jlengstorf Not all new, just a little realign :D (also FWIW not all images are new, I tried to balance out old/new amount, and plan on still removing a bunch more)

@amberleyromo I'll give it a shot because I also want to use `styled` a little more. Btw.…already got annoyed a couple times _having_ to name things again ;)